paysource_test.go

```diff
@@ -15,8 +15,6 @@ func TestPaymentType(t *testing.T) {
 	is.Equal("mandiri_clickpay", midtrans.SourceMandiriClickpay)
```

Comment:
@haritsfahreza great contribution 👍 
One step away from perfect,
How about adding `gopay` & `danamon_online` to the `paysource_test.go`?

Reply:
Done @rizdaprasetya please take a look 🙏